editor/block-toolbar/index.js

```diff
@@ -40,6 +76,54 @@ class BlockToolbar extends Component {
 		} ) );
 	}
 
+	onKeyDown( event ) {
+		if ( metaKeyPressed( event ) ) {
+			this.metaCount++;
+		}
+	}
+
+	stopPropagation( event ) {
+		// This is required to avoid messing up with the WritingFlow navigation
+		event.stopPropagation();
+	}
+
+	onKeyUp( event ) {
+		const isMeta = this.metaCount === 1;
+		this.metaCount = 0;
+
+		// Is there a better way to focus the selected block
+		const selectedBlock = document.querySelector( '.editor-visual-editor__block.is-selected' );
```

Comment:
Hm, I don't really like this, together with the global listeners... What if this is done in the block component, together with the listeners? After all we should only do this inside a block, if a block is selected?

Reply:
I do think this is the right place for all this. It's specific to the block toolbar. But at the same time we don't know where the block toolbar is placed. It's now inside the block but it won't be anymore with #2983. also, these concerns are partially addressed with 6eec643